guides/introduction/welcome.md

```diff
@@ -214,7 +216,9 @@ Phoenix v1.6 and later includes code generators for LiveView. If you want to see
 an example of how to structure your application, from the database all the way up
 to LiveViews, run the following:
 
-    mix phx.gen.live Blog Post posts title:string body:text
+```shell
+$ mix phx.gen.live Blog Post posts title:string body:text
```

Comment:
Perhaps no need in `$` again not to break highlighting?

Reply:
The $ is treated as unselectable by ExDoc and indicates that this is a shell command.